operator/cmd/mesh/install.go

```diff
@@ -182,6 +189,17 @@ func runApplyCmd(cmd *cobra.Command, rootArgs *rootArgs, iArgs *installArgs, log
 		return fmt.Errorf("failed to install manifests: %v", err)
 	}
 
+	// Workaround for broken validation with revisions (https://github.com/istio/istio/issues/28880)
+	// TODO(Monkeyanator) remove once we have a nicer solution
+	if iArgs.revision != "" {
+		if !requiresIstiodServiceCreation(clientset, name.IstioDefaultNamespace) {
+			if err := createIstiodService(clientset, name.IstioDefaultNamespace, iArgs.revision); err != nil {
+				warning := "Did not find existing istiod service and failed to create one."
```

Comment:
Anything useful we can suggest users to do here?  Ask them to manually create the Istiod service?

Reply:
Changed it to better describe the problem and link to our canary upgrade docs which will outline how to create the service manually.

Reply:
Thanks!